### PR TITLE
Added capability to preview dashboard on Tizen player's resolution

### PR DESCRIPTION
**Pull request by Danialkafeel**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--dx-recognitions--hlxscreens.hlx.live/content/screens/org-amitabh/dashboards/ban2-08-130
- After: https://feat-preview-tizen--dx-recognitions--hlxscreens.hlx.live/content/screens/org-amitabh/dashboards/ban2-08-130

### CODE REVIEW

File: tools/preview-dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function adjustScale() {
+  const targetWidth = 1920; // Target resolution width
```
Comment (dhavkuma) [inline tools/preview-dashboard/index.js:2]: lets change constant names to all caps.

File: tools/preview-dashboard/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      min-width: 1920px;
+      min-height: 1080px;
+      border: 1px solid black; /* Optional: visualize the viewport */
+      box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3); /* Optional: enhance appearance */
+    }
+  </style>
+</head>
+<body>
+  <!-- Simulated viewport -->
+  <div id="viewport">
```
Comment (dhavkuma) [inline tools/preview-dashboard/index.html:29]: Rename id to `signage-viewport`

File: tools/preview-dashboard/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  const viewport = document.getElementById('viewport');
+  viewport.style.transform = `scale(${scale})`;
+}
+
+// Adjust scale on load and when the window is resized
+window.addEventListener('load', adjustScale);
+window.addEventListener('load', () => {
+  // load the dashboard
+  const dashboardURL = new URL(window.location.href).searchParams.get('dashboardURL');
+  document.querySelector('#viewport iframe').src = dashboardURL;
```
Comment (dhavkuma) [inline tools/preview-dashboard/index.js:23]: lets not use word `dashboardURL`. rather lets use `url`

File: tools/sidekick/config.json [hunk 4]
```diff
@@ -14,7 +14,15 @@
       "event": "publish-channel",
       "passConfig": true,
       "passReferrer": true,
-      "environments": ["preview"]
+      "environments": ["preview"],
+      "excludePaths": [ "**/tools/preview-dashboard/**" ]
```
Comment (dhavkuma) [inline tools/sidekick/config.json:18]: rename to `preivew-signage`

Comment (dhavkuma) [reply tools/sidekick/config.json:18]: `preview-for-signage`

File: tools/preview-dashboard/index.js (hunk 1 above)
Comment (Danialkafeel) [reply tools/preview-dashboard/index.js]: Done

File: tools/preview-dashboard/index.html (hunk 2 above)
Comment (Danialkafeel) [reply tools/preview-dashboard/index.html]: Done

File: tools/sidekick/config.json (hunk 4 above)
Comment (Danialkafeel) [reply tools/sidekick/config.json]: Done

**Pull request merged**